app/resources/api/v2/shared_behaviour/receptacle.rb

```diff
@@ -38,6 +38,8 @@ module Api::V2::SharedBehaviour::Receptacle
     # Attributes
     attribute :uuid, readonly: true
     attribute :name, delegate: :display_name, readonly: true
+
+    # TODO: these can be removed later, moved onto request_metadata
```

Comment:
What needs to be done here? Data migration? Worth creating a story for it?

Reply:
There are 4 receptacle columns that could be removed.
The data I guess could be moved into the library requests, but the store and request metadata attributes are specific to the new DilutionAndCleanup request which will not exist for those old runs.
Will make a backlog story.

Reply:
Hmm I'm not sure we should remove columns if there's nowhere to put the existing data?